<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlWid</class>
 <widget class="QFrame" name="ControlWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>247</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="8,0,2,2,0,2,0">
   <item>
    <widget class="RingsProgressbar" name="widgetBar" native="true"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>29</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,5,1">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="ScrollLabel" name="labelSongInfo">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="PlayModeLabel" name="label">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../image.qrc">:/image/btn_listOnce.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QToolButton" name="toolButtonLast">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../image.qrc">
         <normaloff>:/image/last_player.png</normaloff>:/image/last_player.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonPlay">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../image.qrc">
         <normaloff>:/image/icon_player.png</normaloff>
         <normalon>:/image/icon_player.png</normalon>:/image/icon_player.png</iconset>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonNext">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../image.qrc">
         <normaloff>:/image/next_player.png</normaloff>:/image/next_player.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>29</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RingsProgressbar</class>
   <extends>QWidget</extends>
   <header>RingsProgressbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ScrollLabel</class>
   <extends>QLabel</extends>
   <header>ScrollLabel.h</header>
  </customwidget>
  <customwidget>
   <class>PlayModeLabel</class>
   <extends>QLabel</extends>
   <header>PlayModeLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../image.qrc"/>
 </resources>
 <connections/>
</ui>
